kotlinx-coroutines-debug/src/module-info.java

```diff
@@ -0,0 +1,16 @@
+module kotlinx.coroutines.debug {
+    requires java.management;
+    requires java.instrument;
+    requires kotlin.stdlib;
+    requires kotlinx.coroutines.core;
+    requires net.bytebuddy;
+    requires net.bytebuddy.agent;
```

Comment:
Is it going to work with our ByteBuddy shadowing?
https://github.com/Kotlin/kotlinx.coroutines/blob/master/kotlinx-coroutines-debug/build.gradle#L40